### PR TITLE
Updated config changes, added cache

### PR DESCRIPTION
**Pull request by virendrapatidar**
Fix include

- Config / Scheduling changes
- Added node cache
- Updates gasPrice logic for flashbots
- Increased tip by 1 gwei for Miners.
- Increased retry if API call result in error.

### CODE REVIEW

**Review by jcvernaleo — DISMISSED** [review]
OK

File: src/ethers/eth.js [hunk 1]
```diff
@@ -53,6 +58,17 @@ async function getWallet(isFlashbotWallet = false) {
...
 }
 
+function getFlashBotGasPrice() {
+  return getProvider()
+    .getFeeData()
+    .then(function (feeData) {
+      const gasPrice = feeData.gasPrice.gt(BN.from(config.vesper.maxTxnGasPrice))
+        ? config.vesper.maxTxnGasPrice
+        : feeData.gasPrice
+      return BN.from(gasPrice).mul(BN.from(125)).div(100).toString()
```
Comment (rokso) [inline src/ethers/eth.js:68]: Ethers BN support operation on numbers and string. 
```suggestion
      const gasPrice = feeData.gasPrice.gt(config.vesper.maxTxnGasPrice)
        ? config.vesper.maxTxnGasPrice
        : feeData.gasPrice
      return BN.from(gasPrice).mul(125).div(100).toString()
```

File: src/service/ops/resurface.js [hunk 2]
```diff
@@ -72,7 +72,8 @@ async function before(data) {
 }
 
 async function after(data) {
-  return require('./pool').after(data)
+  // return require('./pool').after(data)
```
Comment (rokso) [inline src/service/ops/resurface.js:75]: Forgot to remove this?

File: src/controller/.routes.yml [hunk 3]
```diff
@@ -51,48 +47,48 @@ rebalance:
...
           pools: "vUNI"
 
-    - schedule:
-        rate: cron(30 3-6,12-18 ? * * *)
-        input:
-          pools: "vaDAI"
+    # - schedule:
+    #     rate: cron(30 3-6,14-18 ? * * *)
+    #     input:
+    #       pools: "vaDAI"
```
Comment (rokso) [inline src/controller/.routes.yml:62]: Better to delete it. 2 more commented schedules down in this file

File: src/service/ops/resurface.js (hunk 2 above)
Comment (virendrapatidar) [reply src/service/ops/resurface.js:75]: The resurface operation is commented. Raising warning and email alert instead.

File: src/controller/.routes.yml (hunk 3 above)
Comment (virendrapatidar) [reply src/controller/.routes.yml:62]: Aggressive pools ops are manually handled for now. Will be enabled again so commenting for now.

**Review by patidarmanoj10 — DISMISSED** [review]

File: src/ethers/eth.js [hunk 4]
```diff
@@ -53,6 +58,17 @@ async function getWallet(isFlashbotWallet = false) {
...
 }
 
+function getFlashBotGasPrice() {
+  return getProvider()
+    .getFeeData()
+    .then(function (feeData) {
+      const gasPrice = feeData.gasPrice.gt(config.vesper.maxTxnGasPrice)
+        ? config.vesper.maxTxnGasPrice
+        : feeData.gasPrice
+      return BN.from(gasPrice).mul(BN.from(125)).div(100).toString()
```
Comment (rokso) [inline src/ethers/eth.js:68]: Minor
```suggestion
      return BN.from(gasPrice).mul(125).div(100).toString()
```

**Review by rokso — APPROVED** [review]

**Review by patidarmanoj10 — APPROVED** [review]

**Pull request merged**